core/storage/suggestion/gae_models.py

```diff
@@ -3427,3 +3429,95 @@ def export_data(
                     model.last_contribution_date.isoformat())
             }
         return user_data
+
+
+class TranslationCoordinatorsModel(base_models.BaseModel):
+    """Storage model for rights related to translation coordinator.
+
+    The id of each instance is the id of the corresponding language.
+    The ISO 639-1 language code.
+    """
+
+    # The user_ids of the coordinators of this language.
+    coordinator_ids = datastore_services.StringProperty(
+        indexed=True, repeated=True)
+
+    # The number of coordinators of this language. This property is added to
+    # enable the sorting of datastore query results. It is equal to the
+    # length of the coordinator_ids field.
+    # TODO(#18762): Validation for TranslationCoordinatorsModel:
```

Comment:
TODO(#18762): Add a ...

(TODOs should always be instructions.)

Reply:
Done.